### ISSUE TITLE
HTMLElements and SVG elements move in different directions

### ISSUE THREAD

**Issue by davidbcoulson** [anvaka/panzoom#270]
Thank you for the amazing library! I have a combination of HTMLElements and SVG elements and I am finding that when I use the zoom both types of elements seem to move away from each other, breaking their original grouping display.

when it loads (HTMLElements are the cards SVG are the arrows)
![image](https://user-images.githubusercontent.com/10970699/148654443-b5c317b7-edd6-4fae-9c6c-55f2e6e576bf.png)

when I zoom out the HTML elements seem to go backward which is great but the SVG elements seem to also move backward and left. This is causing

![image](https://user-images.githubusercontent.com/10970699/148654533-363767ac-6fae-4e7e-b4dd-f77d4e3e13a0.png)

Please let me know if I missed something  Here is my code as well to help.

`  setPanZoom() {
    var element = document.getElementById(this.workingLevel.toString())
    panzoom(element)
    var everyChild = document.querySelectorAll("svg.leader-line");
    everyChild.forEach(x => {
      panzoom(x)
    })
  }
`

**Comment by anvaka** [comment]
Looks like something specific to they way this is rendered - can you please share a minimal jsbin/jsfiddle to reproduce?

**Comment by davidbcoulson** [comment]
@anvaka you are right    I am using LeaderLine and just found out they are rendered over the page.  I will need to hack it some way else thank you for the quick response.
